.prettierignore

```diff
@@ -0,0 +1,24 @@
+## extensions
+*.*
+!*.css
+!*.js
+!*.json
+!*.less
+!*.md
+!*.ts
+!*.yaml
+!*.yml
+
+
+## same as in .gitignore
+.DS_Store
+coverage
+node_modules
+out
+test.js
+
+
+## special cases
+dependencies/**
+!dependencies/README.md
+styles
```

Comment:
I excluded the entire styles directory, but I'm wondering if we still want to keep `styles/**/src` as formatted? @shd101wyy WDYT?

Reply:
Sure it looks good